app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, file1, file2 = command.split()
```

Comment:
as well you need to check that the splitted command consists of three parts  before unpacking